### DISCUSSION TITLE
How to enable key repeat for vscode.dev (macos)

### DISCUSSION THREAD

**Discussion by alk-mandrianarijaona** [VSCodeVim/Vim#7232] [Q&A]
I am trying to use vscodevim on https://vscode.dev but I can't have my key repeat work properly. For example. holding `j` would not keep going down

**Comment by J-Fields** [comment]
See https://github.com/VSCodeVim/Vim#mac

**Reply by alk-mandrianarijaona** [reply]
It's not exactly the same thing here because I would need to set `ApplePressAndHoldEnabled` for Chrome (or any other browser) instead of `com.microsoft.VSCode`

**Reply by Janitor99** [reply]
Similar thing happened to me for some reason key repeat only worked in the vscode terminal even after I reloaded so I just enabled key repeat on everything by running this command: defaults write NSGlobalDomain ApplePressAndHoldEnabled -bool false

**Reply by nudge-sangbeomhan** [reply]
Same symptom here at github.dev. And command below did help me :)
```defaults write NSGlobalDomain ApplePressAndHoldEnabled -bool false```

**Reply by spephton** [reply]
dude I don't know why this one works and all the other ones (including `defaults write -g`) don't but thank you, this was driving me batty

**Reply by daolanfler** [reply]
Seems that you have to restart your chrome to take effect

**Reply by jbedworth** [reply]
I keep getting zsh: command not found: defaults

**Comment by iuzoong** [comment]
`defaults write -app ~/Applications/Chrome\ Apps.localized/Visual\ Studio\ Code.app ApplePressAndHoldEnabled -bool false`
